core/debug/inc/debug_exports.h

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2016, ARM Limited, All Rights Reserved
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef __DEBUG_EXPORTS_H__
+#define __DEBUG_EXPORTS_H__
+
+#include <stdint.h>
+#include "uvisor_exports.h"
+
+/* Debug box driver -- Version 0
+ * A constant instance of this struct must be instantiated by the unprivileged
+ * code to setup a debug box.*/
+typedef struct TUvisorDebugDriver {
+    void *get_version;
+    void *halt_error;
```

Comment:
You're right. They also help with the driver registration (to check signatures). See also the updated [uvisor-helloworld example](https://github.com/ARMmbed/uvisor-helloworld/pull/60/files#diff-59b37b296cf946ab6951aba2ce0bee36R63).